python/lsst/pipe/tasks/computeExposureSummaryStats.py

```diff
@@ -418,14 +468,18 @@ def update_psf_stats(
         summary.psfStarDeltaSizeScatter = float(psfStarDeltaSizeScatter)
         summary.psfStarScaledDeltaSizeScatter = float(psfStarScaledDeltaSizeScatter)
 
-        if image_mask is not None:
-            maxDistToNearestPsf = maximum_nearest_psf_distance(
-                image_mask,
-                psf_cat,
-                sampling=self.config.psfSampling,
-                bad_mask_bits=self.config.psfBadMaskPlanes
-            )
-            summary.maxDistToNearestPsf = float(maxDistToNearestPsf)
+        if self.config.psfSampling is None:
+            self.log.info("Note: not computing grid-based maxDistToNearestPsf model fidelity. Setting "
+                          "it to NaN.")
```

Comment:
No need to say "setting it to NaN", that's the default value of all non-computed things.